src/Scenes/CheckLocation.fs

```diff
@@ -20,7 +20,7 @@ type Status =
 | Changed
 | Error of string
 
-type Model =
+type [<Pojo>] Model =
```

Comment:
is this really needed? that would mean I have to change my domain model completely